src/BestBooks.js

```diff
@@ -1,30 +1,49 @@
-import React from 'react';
+import axios from "axios";
+import React from "react";
+import BookCarousel from './Carousel.js';
+// import Carousel from 'react-bootstrap/Carousel';
+require("dotenv").config();
```

Comment:
Really nice work so far on this!

I would check with JB on this line here:

`require()` of course comes from Node.js as it uses CommonJS module system. I would consider using ES5/ES6 syntax for imports as this is more common in React.

You can still use an .env file in React without using require(). You can create an `env.development.local` file and store necessary keys there: `REACT_APP_NAME_OF_KEY=keyhere`

- Then, to use the keys, access them using `process.env.REACT_APP_NAME_OF_KEY`

[Custom env variables in React](https://create-react-app.dev/docs/adding-custom-environment-variables/)